WooCommerce/Classes/POS/Presentation/Item Selector/GhostItemCardView.swift

```diff
@@ -3,46 +3,102 @@ import SwiftUI
 struct GhostItemCardView: View {
     @ScaledMetric private var scale: CGFloat = 1.0
     @State private var viewWidth: CGFloat = 0.0
+    private let configuration: GhostItemCardViewConfiguration
+    private let accessory: AnyView?
```

Comment:
`AnyView` enabled a nicer callsite experience (either pass an accessory or don't) but I couldn't find a way to do that with generics, unfortunately.

I think this is the best way, but happy to hear of any other options that we have...